### PR TITLE
Initial frontend elements

### PR DESCRIPTION
**Pull request by teemutontti**
### Changes
- Uses a simple theme with color variables taken from the Figma design
- Added placeholder buttons and search bar on top of the Leaflet map

### CODE REVIEW

File: app/components/map/_MapComponent.client.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ import { Icon } from "leaflet";
 import markerIcon from "leaflet/dist/images/marker-icon.png";
 import markerShadow from "leaflet/dist/images/marker-shadow.png";
 import markerIcon2x from "leaflet/dist/images/marker-icon-2x.png";
+import Controls from "../controls/Controls";
```
Comment (kuukkanen) [inline app/components/map/_MapComponent.client.tsx:23]: Maybe this should be absolute and not relative path?

File: package.json [hunk 2]
```diff
@@ -15,7 +15,11 @@
     "commitlint": "commitlint --edit"
   },
   "dependencies": {
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
```
Comment (kuukkanen) [inline package.json:19]: Do we need Emotion; MUI seems to have styled components built-in. See https://mui.com/system/styled/.

**Review by kuukkanen — COMMENTED** [review]
Look into MUI's own styled components. https://mui.com/system/styled/

**Comment by teemutontti** [comment]
> Look into MUI's own styled components. https://mui.com/system/styled/

Good call! I'd been avoiding styled components since I wasn't fully aware of the benefits, but after reading up, I can see it's definitely the better choice. Thanks, I'll change it up! 😊

**Comment by teemutontti** [comment]
All suggested changes are now done! ✨

File: app/root.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { ThemeProvider } from "@emotion/react";
```
Comment (kuukkanen) [inline app/root.tsx:1]: This should probably be from "@mui/material" and not from Emotion.

**Review by kuukkanen — CHANGES_REQUESTED** [review]

File: app/root.tsx (hunk 3 above)
Comment (teemutontti) [reply app/root.tsx]: Good eye! Fixed! 👍

**Review by kuukkanen — APPROVED** [review]
Seems to work, great job! 👍🏻

**Pull request merged**